### PR TITLE
Add new API to reauthorize existing subscriptions of a user

### PR DESCRIPTION
**Pull request by dergraf**

### CODE REVIEW

File: src/vernemq_dev_api.erl [hunk 1]
```diff
@@ -35,6 +36,53 @@ disconnect_by_subscriber_id(SubscriberId, Opts) ->
             vmq_queue:force_disconnect(QueuePid, normal, proplists:get_bool(do_cleanup, Opts))
     end.
 
+%% @doc Reauthorize subscriptions by username and subscriber_id
+%%
+%% Given a username and subscriber_id all subscriptions
+%% of a matching client are reauthorized against the
+%% currently installed `auth_on_subscribe` hooks.
```
Comment (larshesel) [inline src/vernemq_dev_api.erl]: Clarify that this also holds for the `_m5` hooks?

**Review by larshesel — CHANGES_REQUESTED** [review]
Looks good to me, but would it be possible to add a small testcase in the `vernemq` repo for this?

File: src/vernemq_dev_api.erl (hunk 1 above)
Comment (dergraf) [reply src/vernemq_dev_api.erl]: Fixed

**Comment by dergraf** [comment]
Regarding test cases. I've planned to implement the new API also in vmq_diversity and test the behaviour there, similar to the disconnect API.

**Review by larshesel — APPROVED** [review]

**Pull request merged**